runtime/src/main/java/org/corfudb/runtime/view/AddressSpaceView.java

```diff
@@ -99,6 +103,11 @@ public void resetCaches() {
      * @throws WrongEpochException  If the token epoch is invalid.
      */
     public void write(IToken token, Object data) throws OverwriteException {
+        write(token, data, null);
+    }
+
+    public void write(IToken token, Object data, @Nullable AddressSpaceOptions opts)
+            throws OverwriteException {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the declaration of thrown exception 'org.corfudb.runtime.exceptions.OverwriteException' which is a runtime exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)